Added support for endorsed responses (answers)

miankhalid opened the pull request:
Fixes: https://openedx.atlassian.net/browse/MA-1186

Updated UI for Question type thread:

Outdated | Updated
------------ | -------------
<img src="https://cloud.githubusercontent.com/assets/1710804/12449028/21837dc2-bf9c-11e5-9b21-5059d7dbad60.png" width="250" /> | <img src="https://cloud.githubusercontent.com/assets/1710804/12816893/d915f9bc-cb70-11e5-966c-ad0495d71c11.png" width="250" />

@1zaman @bguertin @BenjiLee plz review

miankhalid commented on VideoLocker/res/layout/discussion_responses_response_row.xml:
```diff
@@ -25,22 +25,33 @@
+            <TextView
+                android:id="@+id/discussion_responses_answer_author_text_view"
+                style="@style/discussion_responses_small_text"
+                android:text="Marked as answer 2 days ago by edxKhalid TA" />
```
yeah i know, this needs to be `tools:text`

bguertin replied:
So fix it? :smile:

miankhalid replied:
was waiting for more review changes, so, I could fix em all in one pass :wink:

bguertin commented:
:+1: Looks good. Could you just add a few pixels of padding between the check mark and the word "Answer"? If you look at iOS you can see there's a little space between. Probably like `4sp` or so.

miankhalid commented:
@bguertin i've updated the screenshots in my first comment.

Also, I moved the drawable addition code to constructor instead of the binding function as an optimization.

bguertin commented:
:+1:

BenjiLee commented on VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java:
```diff
@@ -282,6 +290,21 @@ public void onException(Exception ex) {
+                    R.string.answer_author_attribution, comment, new Runnable() {
+                        @Override
+                        public void run() {
+                            listener.onClickAuthor(comment.getAuthor());
```
Isn't `onClickAuthor` already set?

BenjiLee commented:
:+1:
